views/js/runner/plugins/navigation/button.tpl

```diff
@@ -3,4 +3,5 @@
         <span class="icon-{{icon}}"></span>
         <span class="text">{{text}}</span>
     </a>
+    {{{content}}}
```

Comment:
I think this is confusing as part of the buttonTpl. For this plugin I would even prefer to have the content of `commentTpl` injected with jQuery.